src/pages/Orders.js

```diff
@@ -1,44 +1,70 @@
-import styled from 'styled-components';
+import styled, {css} from 'styled-components';
 
 import Header from '../components/Header/Header.js';
 import Navigation from '../components/Navigation/Navigation';
+import useStoreLogin from '../hooks/useStore_login';
 import useStoreOrder from '../hooks/useStore_order';
 
 export default function Orders() {
   const {orders} = useStoreOrder(state => ({
     orders: state.orders,
   }));
 
+  const {login, setToken} = useStoreLogin(state => ({
+    login: state.login,
+    setToken: state.setToken,
+  }));
+
+  function logout() {
+    setToken(undefined);
+  }
+
   return (
     <>
       <Header />
+      {login.token ? (
+        <UserComponent>
+          <Greet>
+            <span>Hello:</span>
+            <span>{login.credentials.username}</span>
+          </Greet>
+          <LogoutButton onClick={logout}>Logout</LogoutButton>
+        </UserComponent>
+      ) : (
+        <>
+          <SpeechBubbleTipps>
+            You have not logged in. After login, you can receive messages of coupons and new dishes, reserve tables,
```

Comment:
I would wrap a more specific and semantic element around the notifications text e.g. `<p>` ... you might even come up with something smarter than my suggestion 🫣